tests/integration/multipass/test_installer.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import shutil
+
+from craft_providers import multipass
+
+
+def test_is_installed():
+    expected = shutil.which("multipass") is not None
+
+    assert multipass.is_installed() is expected
+
+
+def test_install(uninstalled_multipass):  # pylint: disable=unused-argument
+    assert multipass.is_installed() is False
+
+    multipass_version = multipass.install()
```

Comment:
This really installs multipass in the system as part of the test?

Wouldn't this mess with the system too much?

Reply:
this runs under spread by default... usually all our integration tests are dangerous on native systems which is why we had them completely separate (as spread tasks)

Reply:
No.  It's an integration test, how else are we supposed to get automated coverage for it? :P

And for safety, it's only uninstalled if you specifically set the envvar to do so (CRAFT_PROVIDERS_TESTS_ENABLE_MULTIPASS_UNINSTALL=1)